tests/providers/google/cloud/hooks/test_gcs.py

```diff
@@ -503,7 +503,7 @@ def test_delete_bucket(self, mock_service):
 
         self.gcs_hook.delete_bucket(bucket_name=test_bucket)
 
-        mock_service.return_value.bucket.assert_called_once_with(test_bucket)
+        mock_service.return_value.bucket.assert_called_once_with(test_bucket, user_project=None)
```

Comment:
Is this necessary? It is the default value

Reply:
Yeah I thought it was a little silly, but the assertion method recognizes the difference between specifying `None` upfront and leaving it as the default value. I only updated this code because [CI tests were failing](https://github.com/apache/airflow/actions/runs/5629458795/job/15254477215).